src/styles/mixins.scss

```diff
@@ -0,0 +1,18 @@
+@mixin animation($duration, $timing-function) {
+  @keyframes move {
+    from {
+      transform: rotate(0);
+      transform-origin: $default-transform-origin;
+    }
+
+    to {
+      transform: rotate(360deg);
+      transform-origin: $default-transform-origin;
```

Comment:
The variable `$default-transform-origin` is used in the `transform-origin` property but is not defined within this file. Ensure that `$default-transform-origin` is defined globally or imported from another SCSS file before using it in this mixin.